From 0fee4b1316e3be7e2c8a0f675931acf0d0d36f7a Mon Sep 17 00:00:00 2001
From: Ivan Vecera <ivecera@redhat.com>
Date: Wed, 31 May 2017 15:45:01 +0200
Subject: [PATCH 167/169] tc: man: matchall: Fix example indentation

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1435624
Upstream status: iproute2.git commit d65a744cdbe97

commit d65a744cdbe973cf807be108c73c91c0996b9c66
Author: Yotam Gigi <yotamg@mellanox.com>
Date:   Tue Jan 31 11:47:47 2017 +0200

    tc: man: matchall: Fix example indentation

    The man page contains two examples, which have different indentation. Fix
    the indentation of the two examples to match.

    Reviewed-by: Jiri Pirko <jiri@mellanox.com>
    Signed-off-by: Yotam Gigi <yotamg@mellanox.com>

Signed-off-by: Ivan Vecera <ivecera@redhat.com>
---
 man/man8/tc-matchall.8 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/man/man8/tc-matchall.8 b/man/man8/tc-matchall.8
index f9209226..5aa11dab 100644
--- a/man/man8/tc-matchall.8
+++ b/man/man8/tc-matchall.8
@@ -53,6 +53,7 @@ where the second command attaches a matchall filters on it that mirrors the
 packets to device eth2.
 
 To create egress mirroring from port eth1 to port eth2:
+.RS
 .EX
 
 tc qdisc add dev eth1 handle 1: root prio
-- 
2.13.0

